### PR TITLE
CSS Variables

### PR DESCRIPTION
**Pull request by JanLawrence**
Added guide for css -> css-variables

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [ ] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [ ] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

**Comment by RandellDawson** [comment]
Closed and Reopened this PR to attempt to resolve a specific Travis build failure.

File: guide/english/css/css-variables/index.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+title: CSS Variables
+---
+## CSS Custom Properties (Variables)
+The var() function can be used to insert the value of a custom property.
+
+## The var() Function
+Variables in CSS should be declared within a CSS selector that defines its scope. For a global scope you can use either the :root or the body selector.
```
Comment (co16353sidak) [inline guide/english/css/css-variables/index.md]: add code emphasis on :root and on body as `:root` and `body`

**Review by theNaut1lus — CHANGES_REQUESTED** [review]
Added a small change, as such new guide file being created, not able to find any duplicate files, seems viable to merge post rectification.

/cc: @RandellDawson

**Comment by RandellDawson** [comment]
Thank you for opening this pull request.

This is a standard message notifying you that we’ve reviewed your pull request and have decided not to merge it. We would welcome future pull requests from you.

**Reason:**  The changes being added here were taken directly from https://www.w3schools.com/css/css3_variables.asp.  We do not allow plagiarized content in the Guide.

**Pull request closed without merging**